sdk/spring/azure-spring-cloud-test-servicebus-binder/src/test/resources/application-properties.yml

```diff
@@ -0,0 +1,43 @@
+spring:
+  cloud:
+    stream:
+      function:
+        definition: queueConsume;queueSupply;topicConsume;topicSupply;
+      bindings:
+        topicConsume-in-0:
+          destination: topic2
+          group: topicSub
+        topicSupply-out-0:
+          destination: topic2
+        queueConsume-in-0:
+          binder: servicebus-2
+          destination: queue2
+        queueSupply-out-0:
+          binder: servicebus-2
+          destination: queue2
+      binders:
+        servicebus-1:
+          type: servicebus-topic
+          default-candidate: true
+          environment:
```

Comment:
is this added new?
very wired

Reply:
Yes, in order to add an integration test for the configuration of the added `serviceBusReceiveMode` parameter, I can delete it if it is not necessary😅

Reply:
never mind, I was wrong, just asked Yi Liu about this